TODO

```diff
@@ -2,11 +2,38 @@ We moved TODO list to our Wiki. You may find it at the following link:
 
```

Comment:
Typically git commit messages tell what the commit _does_ upon being applied, so they look like "Add hardcoded reverse iterator".